streamer/src/nonblocking/stream_throttle.rs

```diff
@@ -21,22 +21,47 @@ const STREAM_THROTTLING_INTERVAL_MS: u64 = 100;
 pub const STREAM_STOP_CODE_THROTTLING: u32 = 15;
 const STREAM_LOAD_EMA_INTERVAL_MS: u64 = 5;
 const STREAM_LOAD_EMA_INTERVAL_COUNT: u64 = 10;
-const MIN_STREAMS_PER_THROTTLING_INTERVAL_FOR_STAKED_CONNECTION: u64 = 8;
+const EMA_WINDOW_MS: u64 = STREAM_LOAD_EMA_INTERVAL_MS * STREAM_LOAD_EMA_INTERVAL_COUNT;
 
 pub(crate) struct StakedStreamLoadEMA {
     current_load_ema: AtomicU64,
     load_in_recent_interval: AtomicU64,
     last_update: RwLock<Instant>,
     stats: Arc<StreamStats>,
+    max_staked_load_in_ema_window: u64,
+    max_unstaked_load_in_throttling_window: u64,
```

Comment:
I think we can use some documentation to explain these two fields -- especially because the first one is the aggregate for all staked connections and the second is for that of the per unstaked connection.